lib/segment/src/index/hnsw_index/graph_links.rs

```diff
@@ -52,11 +56,15 @@ const HEADER_SIZE: usize = 64;
 
 #[derive(Clone, Debug, Default)]
 struct GraphLinksFileInfo {
+    compressed: bool,
+    m: usize,
+    m0: usize,
     point_count: usize,
     reindex_start: usize,
     links_start: usize,
     offsets_start: usize,
     offsets_end: usize,
+    bits_per_unsorted: u8,
 }
 
 #[derive(AsBytes, FromBytes, FromZeroes)]
```

Comment:
Can we _just_ extend the header type (and size) here without problems?

Reply:
Yes, we can. We have `HEADER_SIZE` constant which is larger that actual size of this structure. It allows us to extend this file. Only one thing to check. New fields will be zeroes in old files